script/Arcueid/Day 8/08_00_ARC08_6B.txt

```diff
@@ -826,17 +914,19 @@ Despite its surreal appearance, I took its life without hesitating.
 [sha:eecce3cca4e0055592f1be7c9b6e1bb1d047493e]{
 -- Page 404, Offset 20711.
 -- あたまが、うまく思考してくれない。
-My head cannot think properly.
+I can't think clearly.
 }
 [sha:a803f28c773aabdd5123007b60965ea9a55393b8]{
 -- Page 404, Offset 20712.
 -- 今のは、ただの暴力だった。
-That was simply plain violence just now.
+That was nothing but mindless violence just now.
 }
 [sha:a996ba4eff48459613cf0641408ea5ae2cb87b57]{
 -- Page 404, Offset 20713.
 -- 先生が教えてくれた教訓を一部も含めない、<感情|いしき>にまかせた暴走だった。そうなってはいけないと、ずっと恐れを抱いていた殺人鬼の行為だった。
-I did not take any of Sensei's lessons into consideration. I simply let my mind go wild. Even though I shouldn't have done that, I acted like the killer that I had always feared.
+I did not take any of Sensei's lessons into consideration.
+ I simply let my <emotions|m i n d   > run wild.
```

Comment:
The reason I brought it back is because "letting one's emotions run wild" is very much a set phrase, and reading it with only "mind" there just makes me think it's _supposed_ to be emotions, which is what's actually there in the subtext

Reply:
That's fine then I guess.